File: 01-FUNDAMENTOS_DE_PROGRAMACAO/Java/Exercicio2.java
/*
2 - Dado um número inteiro positivo de 3 algarismos, inverter a ordem de seus
algarismos. Os três algarismos do número dado são diferentes de zero.
 */

import java.util.Scanner;

public class exercicio2 {
    public static void main(String[] args) {
        Scanner teclado = new Scanner (System.in);
        int centena, dezena, unidade;
        System.out.println("Digite um numero inteiro de 3 algarismos diferente de 0: ");
        int num = teclado.nextInt();
        
        centena = num/100;
        dezena = (num%100)/10;
        unidade = (num%100)%10;
        
        System.out.println("A ordem ivertida é: " +unidade +dezena + centena);
    }
}

File: 01-FUNDAMENTOS_DE_PROGRAMACAO/Python/exercicio2.py
""" 2 - Dado um número inteiro positivo de 3 algarismos, inverter a ordem de seus
algarismos. Os três algarismos do número dado são diferentes de zero. """

num = int(input('Digite um numero inteiro de 3 algarismos diferente de 0: '))

centena = int(num/100)
dezena = int((num%100)/10)
unidade = int((num%100)%10)

print('O numero invertido é: ' +str(unidade) +str(dezena) +str(centena))